Server/src/Server.Api/Controllers/PublishersController.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
-using Server.Application.CommandHandlers;
+using Server.Application.CommandHandlers.Admin;
 using Server.Application.ViewModels;
 using Server.Infrastructure.Persistence.QueryHandlers;
 
 namespace Server.Api.Controllers;
 
 [ApiController]
+[Authorize(Roles = "User")]
```

Comment:
Czemu tu dodana rola, a w innych kontrolerach nie?

Reply:
Czyli w sumie wychodzi, że cały kontroler nie będzie miał chyba [Authorize], bo przeglądanie oferty książek chyba nie wymaga bycia zalogowanym

Reply:
nie powinno w sumie tego usera ani w ogóle authoriza być na kontrolerze masz racje

Reply:
ja bym i tak na razie wywalił to authorize bo to jest tylko jedna adnotacja a dopóki nie ma logowania na froncie zapiętego to będzie to tylko utrudniać testowanie

Reply:
@DontEatRice w sumie racja, to można dodać potem